### PR TITLE
Adding APO docs

### PR DESCRIPTION
**Pull request by dcpena**
Adding Automatic Platform Optimization docs

### CODE REVIEW

File: developers.cloudflare.com/static/robots.txt [hunk 1]
```diff
@@ -1,3 +1,42 @@
...
\ No newline at end of file
+Disallow: /browser-isolation
+
+Sitemap: https://developers.cloudflare.com/developers/sitemap.xml
+Sitemap: https://developers.cloudflare.com/onedot/sitemap.xml
+Sitemap: https://developers.cloudflare.com/access/sitemap.xml
+Sitemap: https://developers.cloudflare.com/analytics/sitemap.xml
+Sitemap: https://developers.cloudflare.com/api/sitemap.xml
+Sitemap: https://developers.cloudflare.com/argo-tunnel/sitemap.xml
+Sitemap: https://developers.cloudflare.com/automatic-platform-optimzation/sitemap.xml
```
Comment (pedrosousa) [inline developers.cloudflare.com/static/robots.txt]:
```suggestion
Sitemap: https://developers.cloudflare.com/automatic-platform-optimization/sitemap.xml
```

**Comment by pedrosousa** [comment]
I'd remove the `.DS_Store` file.

File: README.md [hunk 2]
```diff
@@ -18,6 +18,7 @@ To get write access to this repo, please reach out to the __Developer Docs__ roo
 | Access                  | access               | [Test](https://access.cloudflare-docs.workers.dev/access)                             | [Prod](https://developers.cloudflare.com/access)               |
 | Analytics               | analytics            | [Test](https://analytics.cloudflare-docs.workers.dev/analytics)                       | [Prod](https://developers.cloudflare.com/analytics)            |
 | API                     | api                  | [Test](https://api.cloudflare-docs.workers.dev/api)                                   | [Prod](https://developers.cloudflare.com/api)                  |
+| Automatic Platform Optimization      | automatic-platform-optimization     | [Test](https://automatic-platform-optimization.cloudflare-docs.workers.dev/api)             | [Prod](https://developers.cloudflare.com/automatic-platform-optimization)           |
```
Comment (pedrosousa) [inline README.md]:
```suggestion
| Automatic Platform Optimization | automatic-platform-optimization | [Test](https://automatic-platform-optimization.cloudflare-docs.workers.dev/automatic-platform-optimization) | [Prod](https://developers.cloudflare.com/automatic-platform-optimization) |
```

**Review by YoutacRSVARs — APPROVED** [review]

File: products/automatic-platform-optimization/src/content/get-started/activate-cf-wp-plugin.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
+---
+title: Activate the Cloudflare WordPress plugin
```
Comment (kodster28) [inline products/automatic-platform-optimization/src/content/get-started/activate-cf-wp-plugin.md:2]: For SEO, may want to spell out the abbreviations in the file name, `activate-cloudflare-wordpress-plugin.md`.

File: products/automatic-platform-optimization/docs-config.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    },
+    {
+      title: "Blog: APO Post-launch report",
+      url: "https://blog.cloudflare.com/apo-post-launch-report/"
+    }
+  ],
+  search: {
+    indexName: "developers-cloudflare",
+    apiKey: "b23088ab4d346409f9d3ece6606344c3",
+    algoliaOptions: { 'facetFilters': '["project:automatic-platform-optimization"]'}
```
Comment (kodster28) [inline products/automatic-platform-optimization/docs-config.js:24]: Since you're initializing the search right away, expect about a 2 day delay in it actually working (Algolia has to merge a PR and then it takes about a day to scrape your site).

If that's an issue, you might want to leave the search object blank first, and then make a separate PR once they've merged the PR and had time to scrape your docs.

File: products/automatic-platform-optimization/docs-config.js [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+  },
+  siteMetadata: {
+    title: "Cloudflare Automatic Platform Optimization",
+    description: "With Automatic Platform Optimization, Cloudflare serves your entire site from our edge network, ensuring customers see improved performance when visiting your site.",
```
Comment (kodster28) [inline products/automatic-platform-optimization/docs-config.js:28]: Feel like the last `your site` is redundant?

File: products/automatic-platform-optimization/src/content/about/index.md [hunk 6]
```diff
@@ -0,0 +1,10 @@
+---
+title: About
+order: 1
+---
```
Comment (kodster28) [inline products/automatic-platform-optimization/src/content/about/index.md:4]: Since you're here, would you be able to add the `pcx-content-type` attribute?

File: products/automatic-platform-optimization/src/content/index.md [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+order: 0
+---
+
+# Welcome
+
+Take your WordPress site’s performance to the next level with Automatic Platform Optimizations (APO). APO allows Cloudflare to serve your entire WordPress site from its edge network ensuring consistent, fast performance for visitors no matter where they are.
+
+Automatic Platform Optimization is the result of using the power of Cloudflare Workers to intelligently cache dynamic content. By caching dynamic content, Cloudflare can serve the entire website from our edge network to make a site's time to first byte (TTFB) both fast and consistent.
+
+To read more about the benefits of using APO with your site, see [The Benefits of Automatic Platform Optimization blog](https://blog.cloudflare.com/automatic-platform-optimizations-starting-with-wordpress/#the-benefits-of-automatic-platform-optimization).
```
Comment (kodster28) [inline products/automatic-platform-optimization/src/content/index.md:12]: Not sure your thoughts on this, but I've enjoyed the multiple CTAs at the bottom of our docs pages.

Example in [Firewall](https://developers.cloudflare.com/firewall) docs.

File: products/automatic-platform-optimization/src/content/troubleshooting/index.md [hunk 8]
```diff
@@ -0,0 +1,23 @@
+---
+title: Troubleshooting
+order: 14
+---
+
+# Troubleshooting
+
+## WordPress plugin is undetected on Cloudflare dashboard
+
+The WordPress plugin may go undetected on your Cloudflare dashboard for a few reasons.
```
Comment (kodster28) [inline products/automatic-platform-optimization/src/content/troubleshooting/index.md:10]: Change period to `:`

File: products/automatic-platform-optimization/src/content/troubleshooting/index.md [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+- Version 3.8.2 of the plugin is installed but existing cache plugins return stale responses, for example, without `cf-edge-cache` header. 
+  - **Solution:** Enable APO from the WordPress plugin and purge the cache in the existing cache plugins.
+- WordPress only runs on a subdomain, but WordPress and the WordPress plugin check against the zone's root domain.
+  - **Solution:** For additional information, see [Subdomains and subdirectories](/reference/subdomain-subdirectories)
+
+If your Cloudflare dashboard can't detect the WordPress plugin after trying the solutions above, ensure you completed all of the steps listed in [Activate the Cloudflare WordPress plugin](/get-started/activate-cf-wp-plugin).
+
+## WordPress returns stale content
+
+If WordPress is returning stale content, purge the cache when APO is enabled.
```
Comment (kodster28) [inline products/automatic-platform-optimization/src/content/troubleshooting/index.md:23]: Do you need a **Solution:** here as well?

File: products/automatic-platform-optimization/src/content/troubleshooting/faq.md [hunk 10]
```diff
@@ -0,0 +1,43 @@
+---
+title: FAQs
+order: 15
+---
+
+# Frequently Asked Questions
+
+## Do I still need to create "Edge Cache TTL" page rules with "Cache Level: Cache Everything"?
+
+No, you don't need create Edge Cache TTL page rules. When the WordPress plugin is installed, APO automatically caches content for 30 days and invalidates on change within 30 seconds. However, because APO now supports cache-related page rules, make sure existing page rules don't affect the resources served by APO.
```
Comment (kodster28) [inline products/automatic-platform-optimization/src/content/troubleshooting/faq.md:10]: `Invalidates on` sounds funny to me

File: products/automatic-platform-optimization/src/content/troubleshooting/faq.md [hunk 11]
```diff
@@ -0,0 +1,43 @@
...
+
+## Does Origin Cache Control override APO?
+
+No. APO ignores Origin Cache Control for caching on the Edge, but APO serves original Origin Cache Control to the client. 
+
+## Why are my browser cache control headers missing with APO?
+
+The browser cache control headers may be missing with APO if you set your browser to cache TTL to "respect existing headers." For example:
```
Comment (kodster28) [inline products/automatic-platform-optimization/src/content/troubleshooting/faq.md:18]: Should this be a `code` thing instead of a "quote" thing?

File: products/automatic-platform-optimization/src/content/troubleshooting/faq.md [hunk 12]
```diff
@@ -0,0 +1,43 @@
...
+```
+$ curl -sv -o /dev/null https://example.com/ -H 'Accept: text/html' 2>&1 | grep cache-control
+< cache-control: max-age=86400, stale-while-revalidate=86400, stale-if-error=86400
+```
+
+##  Is the stale-if-error directive still needed with APO?
+
+No, the stale-if-error directive is not needed because the feature is built into APO.
+
+## When I check the posts and homepage cache status, the response header shows `cf-cache-status: BYPASS`. Is APO working?
```
Comment (kodster28) [inline products/automatic-platform-optimization/src/content/troubleshooting/faq.md:29]: You almost got all of them! Think this one should start with a `Yes.`

File: products/automatic-platform-optimization/src/content/troubleshooting/faq.md [hunk 13]
```diff
@@ -0,0 +1,43 @@
+---
+title: FAQs
+order: 15
+---
+
+# Frequently Asked Questions
```
Comment (kodster28) [inline products/automatic-platform-optimization/src/content/troubleshooting/faq.md:6]: Not sure if this is the nature of the beast... but if I'm a WP user and not super sophisticated, this entire section is beyond me.

Like, would I know what a Cache is or what page rules are?

I'd guess this'd be involved in future edits rather than part of this PR, but just thought I'd bring it up.

File: products/automatic-platform-optimization/src/content/about/plugin-compatibility.md [hunk 14]
```diff
@@ -0,0 +1,36 @@
+---
+title: Plugin compatibility
+order: 3
+---
+
+# Plugin compatibility
+
+Currently, WordPress offers over 50,000 plugins for download. As a result, testing the compatibility between APO and every available plugin is impossible. However, Cloudflare has a list of officially supported plugins and a list of plugins known to cause issues when APO is enabled.
+
+For questions about a specific plugin not shown in the list, create a thread in the [Cloudflare Community](https://community.cloudflare.com/) to begin the conversation.
```
Comment (kodster28) [inline products/automatic-platform-optimization/src/content/about/plugin-compatibility.md:10]: Should you send them to a more specific page of the community?
`https://community.cloudflare.com/tag/wordpress`

File: products/automatic-platform-optimization/src/content/about/plugin-compatibility.md [hunk 15]
```diff
@@ -0,0 +1,36 @@
...
+- [YITH WooCommerce Wishlist](https://wordpress.org/plugins/yith-woocommerce-wishlist/)
+- [WP EasyCart](https://wordpress.org/plugins/wp-easycart/)
+- [Ecwid Ecommerce Shopping Cart](https://wordpress.org/plugins/ecwid-shopping-cart/)
+- [WP ECommerce](https://wordpress.org/plugins/wp-e-commerce/)
+- [Bookly](https://wordpress.org/plugins/bookly-responsive-appointment-booking-tool/)
+- [WPTouch](https://wordpress.org/plugins/wptouch/)
+- [Mobile Detect](https://wordpress.org/plugins/tinywp-mobile-detect/)
+- [WordPress Mobile Pack](https://wordpress.org/plugins/wordpress-mobile-pack/)
+- [WP-Mobilizer](https://wordpress.org/plugins/wp-mobilizer/)
+- [WP Mobile Edition](https://wordpress.org/plugins/wp-mobile-edition/)
```
Comment (kodster28) [inline products/automatic-platform-optimization/src/content/about/plugin-compatibility.md:27]: This plugin is deprecated --> https://wordpress.org/plugins/wp-mobile-edition/

File: products/automatic-platform-optimization/src/content/about/plugin-compatibility.md [hunk 16]
```diff
@@ -0,0 +1,36 @@
...
+- [Any Mobile Theme Switcher](https://wordpress.org/plugins/any-mobile-theme-switcher/)
+- [Easy Social Share Buttons](https://codecanyon.net/item/easy-social-share-buttons-for-wordpress/6394476)
+- [Jetpack (Mobile Theme)](https://wordpress.org/plugins/jetpack/)
+- [wiziApp](https://wordpress.org/plugins/wiziapp-create-your-own-native-iphone-app)
```
Comment (kodster28) [inline products/automatic-platform-optimization/src/content/about/plugin-compatibility.md:31]: This plugin is also deprecated --> https://wordpress.org/plugins/wiziapp-create-your-own-native-iphone-app/

File: products/automatic-platform-optimization/src/content/about/plugin-compatibility.md [hunk 17]
```diff
@@ -0,0 +1,36 @@
+---
+title: Plugin compatibility
+order: 3
```
Comment (kodster28) [inline products/automatic-platform-optimization/src/content/about/plugin-compatibility.md:3]: Broader question... do we need all of these listed plugins and links?

There's a pretty high maintenance cost to this page, illustrated by the fact that we already have 2 dead links.

File: products/automatic-platform-optimization/README.md [hunk 18]
```diff
@@ -0,0 +1,5 @@
+# Cloudflare Automatic Platform Optimization docs
```
Comment (kodster28) [inline products/automatic-platform-optimization/README.md:1]: As part of this work, could you make sure that they also update the links on this page?
https://wordpress.org/plugins/cloudflare/

File: products/automatic-platform-optimization/src/content/reference/query-parameters.md [hunk 19]
```diff
@@ -0,0 +1,56 @@
+---
+title: Query parameters and cached responses
+order: 10
+---
+
+# Query parameters and cached responses
+
+Query parameters often signal the presence of dynamic content. As a result, if there are query parameters in the URL, APO bypasses the cache and attempts to get a new version of the page from the origin by default. Because query parameters are also often used for marketing attribution, like UTMs, quick loading times are especially important for users. 
+
+To add a query parameter to our allowlist, [create a post in the community](https://community.cloudflare.com/) for consideration.
```
Comment (kodster28) [inline products/automatic-platform-optimization/src/content/reference/query-parameters.md:10]: More specific community link?

File: products/automatic-platform-optimization/src/content/troubleshooting/faq.md (hunk 10 above)
Comment (kodster28) [inline products/automatic-platform-optimization/src/content/troubleshooting/faq.md:10]: Should you also be linking back to the page rule reference page?

File: products/automatic-platform-optimization/src/content/reference/page-rule-integration.md [hunk 20]
```diff
@@ -0,0 +1,22 @@
...
+
+The following Page rules can control APO. Any changes to Caching page rules require purging the cache for the changes to take effect.
+
+- **Cache Level: Bypass** — APO bypasses pages with response header `cf-apo-via: origin,page-rules`
+- **Cache Level: Ignore Query String** — APO ignores all query strings when serving from Cache.
+- **Cache Level: Cache Everything** — APO caches pages with all query strings. Default cookies rules still apply. 
+
+  <Aside type="warning">
+
+    Automatic page purge via the WordPress plugin won’t clean all cached pages, only pages without query strings. Cached responses will be returned even with request header `cache-control: no-cache`.
```
Comment (kodster28) [inline products/automatic-platform-optimization/src/content/reference/page-rule-integration.md:16]: Will not

File: products/automatic-platform-optimization/src/content/reference/cache-device-type.md [hunk 21]
```diff
@@ -0,0 +1,34 @@
+---
+title: Cache by device type
+order: 12
+---
+
+# Cache by device type
+
+APO cache by device type provides all of the same benefits of Cloudflare's cache while targeting visitors with content appropriate to their device. Cloudflare evaluates the `User-Agent` header in the HTTP request to identify the device type. Cloudflare then identifies each device type with a case insensitive match to the regex below:
```
Comment (kodster28) [inline products/automatic-platform-optimization/src/content/reference/cache-device-type.md:8]: I'd add a line break between the first and second sentences, just since it's a lot of info.

File: products/automatic-platform-optimization/src/content/reference/cache-device-type.md [hunk 22]
```diff
@@ -0,0 +1,34 @@
...
+The Cloudflare for WordPress plugin automatically purges all cache variations for updated pages. If you use any of the plugins listed below, enable cache by device type.
+
+  - [WP Touch (Free Version)](https://wordpress.org/plugins/wptouch/)
+  - [Wp Mobile Detect](https://wordpress.org/plugins/wp-mobile-detect/)
+  - [wiziApp](https://wordpress.org/plugins/wiziapp-create-your-own-native-iphone-app)
+  - [WordPress Mobile Pack](https://wordpress.org/plugins/wordpress-mobile-pack/)
+  - [WP-Mobilizer](https://wordpress.org/plugins/wp-mobilizer/)
+  - [WP Mobile Edition](https://wordpress.org/plugins/wp-mobile-edition/)
+  - [Device Theme Switcher](https://wordpress.org/plugins/device-theme-switcher/)
+  - [Easy Social Sahre Buttons for WordPress](https://codecanyon.net/item/easy-social-share-buttons-for-wordpress/6394476)
```
Comment (kodster28) [inline products/automatic-platform-optimization/src/content/reference/cache-device-type.md:33]: Social `Share`

File: products/automatic-platform-optimization/src/content/reference/cache-device-type.md [hunk 23]
```diff
@@ -0,0 +1,34 @@
...
+  - [Jetpack (when mobile theme is activated)](https://jetpack.com/support/mobile-theme/)
```
Comment (kodster28) [inline products/automatic-platform-optimization/src/content/reference/cache-device-type.md:34]: Mobile theme is deprecated

File: products/automatic-platform-optimization/src/content/reference/cache-device-type.md [hunk 24]
```diff
@@ -0,0 +1,34 @@
...
+
+Changing Cache By Device Type setting will invalidate Cache.
+
+</Aside>
+
+The Cloudflare for WordPress plugin automatically purges all cache variations for updated pages. If you use any of the plugins listed below, enable cache by device type.
+
+  - [WP Touch (Free Version)](https://wordpress.org/plugins/wptouch/)
+  - [Wp Mobile Detect](https://wordpress.org/plugins/wp-mobile-detect/)
+  - [wiziApp](https://wordpress.org/plugins/wiziapp-create-your-own-native-iphone-app)
```
Comment (kodster28) [inline products/automatic-platform-optimization/src/content/reference/cache-device-type.md:28]: deprecated

File: products/automatic-platform-optimization/src/content/reference/cache-device-type.md [hunk 25]
```diff
@@ -0,0 +1,34 @@
...
+  - [WordPress Mobile Pack](https://wordpress.org/plugins/wordpress-mobile-pack/)
+  - [WP-Mobilizer](https://wordpress.org/plugins/wp-mobilizer/)
+  - [WP Mobile Edition](https://wordpress.org/plugins/wp-mobile-edition/)
```
Comment (kodster28) [inline products/automatic-platform-optimization/src/content/reference/cache-device-type.md:31]: deprecated

File: products/automatic-platform-optimization/src/content/get-started/change-nameservers.md [hunk 26]
```diff
@@ -0,0 +1,41 @@
+---
```
Comment (kodster28) [inline products/automatic-platform-optimization/src/content/get-started/change-nameservers.md:1]: More of a longer-term note, but this should probably be an `include` that we share across multiple areas.

File: products/automatic-platform-optimization/src/content/get-started/verify-apo-works.md [hunk 27]
```diff
@@ -0,0 +1,24 @@
...
+# Verify APO works
+
+You can check whether or not APO is working by verifying APO headers are present. When APO is working, three headers are present: `CF-Cache-Status`, `cf-apo-via`, `cf-edge-cache`.
+
+1. Visit [Uptrends.com](https://www.uptrends.com/tools/http-response-header-check).
+1. In the text field, enter the URL for your WordPress homepage including the `https://www.`.
+1. Select **Start test**. The **Response Headers** table displays.
+1. Locate the three header responses and their description. APO is working correctly when the headers exactly match the headers below.
+
+- `CF-Cache-Status` | `HIT`
```
Comment (kodster28) [inline products/automatic-platform-optimization/src/content/get-started/verify-apo-works.md:15]: Inconsistent capitalization on this header.

File: products/automatic-platform-optimization/src/content/get-started/verify-apo-works.md [hunk 28]
```diff
@@ -0,0 +1,24 @@
...
+  - The `cf-cache-status` header displays if the asset is served from the cache or was considered dynamic and served from the origin.
+- `cf-apo-via` | `tcache`
```
Comment (kodster28) [inline products/automatic-platform-optimization/src/content/get-started/verify-apo-works.md:17]: To me, having the name of the header in the description makes it harder to read. Suggest removing since it's in a sub-bullet

File: products/automatic-platform-optimization/src/content/get-started/verify-apo-works.md [hunk 29]
```diff
@@ -0,0 +1,24 @@
...
+  - The `cf-apo-via` header returns the APO status for the given request.
+- `cf-edge-cache` | `cache, platform=wordpress`
+  - The `cf-edge-cache` headers confirms the WordPress plugin is installed and enabled.
+
+## Verify the APO integration and WordPress integration work
+
```
Comment (kodster28) [inline products/automatic-platform-optimization/src/content/get-started/verify-apo-works.md:23]: This section feels like it's just repeating the info above?

File: products/automatic-platform-optimization/src/content/get-started/activate-cf-wp-plugin.md [hunk 30]
```diff
@@ -0,0 +1,65 @@
...
+order: 7
+---
+
+# Activate the Cloudflare WordPress plugin
+
+For users on the free plan, be sure to purchase APO before installing the WordPress plugin. For users on a Pro plan or higher, continue to Install and activate the Cloudflare WordPress plugin.
```
Comment (kodster28) [inline products/automatic-platform-optimization/src/content/get-started/activate-cf-wp-plugin.md:8]: Should you do some anchor links for Purchase and Install?

`[Purchase](#purchase-apo)`

File: products/automatic-platform-optimization/src/content/get-started/activate-cf-wp-plugin.md [hunk 31]
```diff
@@ -0,0 +1,65 @@
...
+## Purchase APO
+
+1. Open your Cloudflare dashboard.
+1. Select the **Speed** tile and locate the **Optimized Delivery** section.
+1. From **Automatic Platform Optimization for WordPress**, select **Purchase**.
+1. Enter your payment information and select **Confirm payment**.
+
+## Install and activate the Cloudflare WordPress plugin
+
+The easiest way to begin using APO is directly from Cloudflare’s WordPress plugin. Before you can use APO, you must first install and activate the plugin and then activate APO.
```
Comment (kodster28) [inline products/automatic-platform-optimization/src/content/get-started/activate-cf-wp-plugin.md:19]: The Cloudflare WordPress plugin. Marketing doesn't like possessives with Cloudflare.

File: products/automatic-platform-optimization/src/content/get-started/activate-cf-wp-plugin.md [hunk 32]
```diff
@@ -0,0 +1,65 @@
...
+
+1. Navigate and log in to your WordPress account.
+1. Select **Plugins** > **Add new**.
+1. In the search field, enter `Cloudflare`.
+1. Locate the Cloudflare plugin and select **Install now**.
+1. After the plugin finishes installing, select **Activate**. The Cloudflare plugin now displays in your Plugins list.
+
+<Aside type="warning">
+
+Cloudflare recommends turning off plugins such as WP Rocket Cache Plugin, W3 Total Cache, or similar plugins when first setting up APO. After confirming APO is working, we recommend testing whether turning on the plugins listed above improves results or causes unexpected behavior. In many cases, using APO along with other caching plugins can cause unexpected results.
```
Comment (kodster28) [inline products/automatic-platform-optimization/src/content/get-started/activate-cf-wp-plugin.md:29]: Get rid of the 2nd `we recommend`. Just `working, test whether...`

File: products/automatic-platform-optimization/src/content/get-started/activate-cf-wp-plugin.md [hunk 33]
```diff
@@ -0,0 +1,65 @@
...
+
+We also recommend clearing the server cache for the WP Rocket Cache plugin, W3 Total Cache, or similar plugins after APO activation.
+
+</Aside>
+
+## Activate APO
+
+To create the connection between WordPress and Cloudflare, you'll create an API token from your Cloudflare dashboard and add it to WordPress. To setup APO on a subdomain, see [Subdomains and subdirectories](/reference/subdomains-subdirectories).
```
Comment (kodster28) [inline products/automatic-platform-optimization/src/content/get-started/activate-cf-wp-plugin.md:37]: Remove the `you'll`

File: products/automatic-platform-optimization/src/content/get-started/activate-cf-wp-plugin.md [hunk 34]
```diff
@@ -0,0 +1,65 @@
...
+
+## Create the API token from Cloudflare
```
Comment (kodster28) [inline products/automatic-platform-optimization/src/content/get-started/activate-cf-wp-plugin.md:39]: Should these be h3's?

File: products/automatic-platform-optimization/src/content/get-started/activate-cf-wp-plugin.md [hunk 35]
```diff
@@ -0,0 +1,65 @@
...
+1. On the **WordPress API token summary** page, select **Create Token**. Your API token displays.
+1. Select the **Copy** button to copy your token. You will need to paste the token in the next section.
+
+<Aside>
+
+Copy and paste your API token into a document saved on your computer to easily reference it again.
+
+</Aside>
+
+## Add your API token to WordPress
```
Comment (kodster28) [inline products/automatic-platform-optimization/src/content/get-started/activate-cf-wp-plugin.md:55]: Should these be h3's?

File: products/automatic-platform-optimization/src/content/get-started/activate-cf-wp-plugin.md [hunk 36]
```diff
@@ -0,0 +1,65 @@
...
+
+## Create the API token from Cloudflare
+
+1. Open your Cloudflare dashboard.
+1. Select **My Profile** from the top of the page. 
+1. Select **API Tokens** > **Create Token**.
+1. Locate **WordPress** from the list and select **Use template**.
+1. Select **Continue to summary** at the bottom of the page.
+1. On the **WordPress API token summary** page, select **Create Token**. Your API token displays.
+1. Select the **Copy** button to copy your token. You will need to paste the token in the next section.
```
Comment (kodster28) [inline products/automatic-platform-optimization/src/content/get-started/activate-cf-wp-plugin.md:47]: remove `to copy your token`

File: products/automatic-platform-optimization/src/content/get-started/activate-cf-wp-plugin.md [hunk 37]
```diff
@@ -0,0 +1,65 @@
...
+
+<Aside>
+
+Copy and paste your API token into a document saved on your computer to easily reference it again.
+
+</Aside>
+
+## Add your API token to WordPress
+
+1. Open your WordPress account and navigate to Plugins.
```
Comment (kodster28) [inline products/automatic-platform-optimization/src/content/get-started/activate-cf-wp-plugin.md:57]: Should **Plugins** be bolded?

File: products/automatic-platform-optimization/src/content/get-started/activate-cf-wp-plugin.md [hunk 38]
```diff
@@ -0,0 +1,65 @@
...
+1. Locate the Cloudflare plugin and select **Settings**.
+1. Select the option to sign in with an existing account.
+1. Enter your email address and paste the token you copied in Step 5 of Create the API token from Cloudflare.
+1. Select **Save API Credentials**.
+1. For **Apply Recommended Cloudflare Settings for WordPress**, select **Apply**.
+1. For **Automatic Platform Optimization**, switch the toggle to **On** to enable APO.
```
Comment (kodster28) [inline products/automatic-platform-optimization/src/content/get-started/activate-cf-wp-plugin.md:63]: Remove `to enable APO`.

**Review by kodster28 — APPROVED** [review]

**Review by YoutacRSVARs — APPROVED** [review]

File: products/automatic-platform-optimization/docs-config.js (hunk 4 above)
Comment (dcpena) [reply products/automatic-platform-optimization/docs-config.js:24]: Thanks for the heads up on that! I don't _think_ it should be an issue so I'll go ahead and leave this as is for now.
